src/commands/junit/README.md

```diff
@@ -22,10 +22,14 @@ datadog-ci junit upload --service my-service --tags key1:value1 --tags key2:valu
 
 - The positional arguments are the directories or file paths in which the jUnit XML reports are located. If you pass a folder, the CLI will look for all `.xml` files in it.
 - `--service` (default: `DD_SERVICE` env var) should be set as the name of the service you're uploading jUnit XML reports for.
-- `--tags` is a array of key value pairs of the shape `key:value`. This will set global tags applied to all spans.
+- `--tags` is an array of key value pairs of the shape `key:value`. This will set global tags applied to all spans.
   - The resulting dictionary will be merged with whatever is in the `DD_TAGS` environment variable. If a `key` appears both in `--tags` and `DD_TAGS`, whatever value is in `DD_TAGS` will take precedence.
 - `--metrics` is an array of key numerical value pairs of the shape `key:123`. This will set global metrics applied to all spans.
   - The resulting dictionary will be merged with whatever is in the `DD_METRICS` environment variable. If a `key` appears both in `--metrics` and `DD_METRICS`, whatever value is in `DD_METRICS` will take precedence.
+- `--report-tags` is an array of key value pairs like the `--tags` argument, but the tags are only applied to the session instead of to every test.
+  - The resulting dictionary will NOT be merged with `DD_TAGS`.
+- `--report-metrics` is an array of key value pairs like the `--metrics` argument, but the tags are only applied to the session instead of to every test.
```

Comment:
```suggestion
- `--report-metrics` is an array of key numerical value pairs like the `--metrics` argument, but the metrics are only applied to the session instead of to every test.
```